src/integrations/Mixpanel/browser.js

```diff
@@ -79,6 +79,7 @@ class Mixpanel {
     this.areTransformationsConnected =
       destinationInfo && destinationInfo.areTransformationsConnected;
     this.destinationId = destinationInfo && destinationInfo.destinationId;
+    this.identityMergeApi = config.identityMergeApi || 'original';
```

Comment:
should we assume this? what if users are using original or simplified?

Reply:
default value of `identityMergeApi` is set to `original` to keep the existing behaviour. Simplified is an new ID merge released by mixpanel 2-3 months ago. Currently this feature is available in mixpanel project with opt-in basis.